x11-terms/aterm/aterm-1.0.1-r5.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic eutils
+
+DESCRIPTION="rxvt compatible terminal emulator with transparency support"
+HOMEPAGE="http://aterm.sourceforge.net"
+SRC_URI="ftp://ftp.afterstep.org/apps/${PN}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64"
+IUSE="background cjk xgetdefault"
+
+RDEPEND="
+	virtual/jpeg:0
+	media-libs/libpng:0=
+	background? ( media-libs/libafterimage )
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libICE
+"
+DEPEND="${RDEPEND}
+	x11-base/xorg-proto
+	x11-libs/libXt
+"
+
+src_prepare() {
+	# Security bug #219746
+	eapply "${FILESDIR}/${P}-display-security-issue.patch"
+	eapply "${FILESDIR}"/${P}-deadkeys.patch
+	eapply "${FILESDIR}/${P}-dpy.patch"
+	eapply "${FILESDIR}/${P}-remove-streams.patch"
+
+	#fix pre-stripped files
+	sed -i -e "/INSTALL_PROGRAM/ s:-s::" autoconf/Make.common.in || die "sed Makefile failed"
+
+	default
+}
+
+src_configure() {
+	local myconf
+
+	use cjk && myconf="$myconf
+		--enable-kanji
+		--enable-thai
+		--enable-big5"
+
+	case "${CHOST}" in
+		*-darwin*) myconf="${myconf} --enable-wtmp" ;;
+		*-interix*) ;;
+		*) myconf="${myconf} --enable-utmp --enable-wtmp"
+	esac
+
+	econf \
+		$(use_enable xgetdefault) \
+		$(use_with background afterimage-config "${EPREFIX}"/usr/bin) \
+		--with-terminfo="${EPREFIX}"/usr/share/terminfo \
+		--enable-transparency \
+		--with-x \
+		${myconf}
+}
+
+src_install() {
+	make DESTDIR="${D}" install || die "make install failed"
```

Comment:
s/make/emake/ and drop the `|| die ...`.

Reply:
Done.